src/main/java/com/minecolonies/coremod/colony/buildings/BuildingLumberjack.java

```diff
@@ -254,6 +267,24 @@ public void serializeToView(@NotNull final ByteBuf buf)
         }
     }
 
+    /**
+     * Whether or not the LJ should replant saplings
+     */
+    public boolean shouldReplant()
+    {
+        return replant;
+    }
+
+    /**
+     * Set whether or not LJ should replant saplings
+     * @param shouldReplant
+     */
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make shouldReplant a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)